Misc/NEWS.d/next/Library/2018-12-14-13-29-17.bpo-35498.LEJHl7.rst

```diff
@@ -0,0 +1 @@
+Add slice support to :meth:`PathLib._PathParents.__getitem__`.
```

Comment:
Better to refer to `:meth:~pathlib.Path.parents`, I believe, since that is the documented function.